src/main/java/org/isf/utils/pagination/PageInfo.java

```diff
@@ -69,4 +64,21 @@ public boolean isHasNextPage() {
 	public void setHasNextPage(boolean hasNextPage) {
 		this.hasNextPage = hasNextPage;
 	}
+
+	public long getTotalNbOfElements() {
+		return totalNbOfElements;
+	}
+
+	public void setTotalNbOfElements(long totalNbOfElements) {
+		this.totalNbOfElements = totalNbOfElements;
+	}
+
+	public long getTotalPage() {
```

Comment:
```suggestion
	public long getTotalPages() {
```
Reads better as plural